components/DropdownLight/DropdownLight.jsx

```diff
@@ -23,20 +35,109 @@ const DropdownSelect = styled.select`
   }}
 `;
 
-const DropdownLight = ({ disabled, items, theme, placeholder }) => (
-  <>
-    <DropdownSelect disabled={disabled} theme={theme} defaultValue="">
-      <option value="" hidden>
-        {placeholder}
-      </option>
-      {items.map(item => (
-        <option value={item?.value || item} key={item?.value || item}>
-          {item?.label || item}
-        </option>
-      ))}
-    </DropdownSelect>
-  </>
-);
+const Input = styled.input`
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  justify-content: space-between;
+  text-align: left;
+`;
+
+const ArrowIcon = styled(Icon)`
+  display: inline-block;
+  pointer-events: none;
+  width: 24px;
+
+  ${({ selectedItem }) =>
+    !selectedItem &&
+    `
+    color: inherit;
+  `}
```

Comment:
The "selectedItem" property was never used below, why is there styling based on it?